bin/hof-transpiler

```diff
@@ -1,53 +1,39 @@
 #!/usr/bin/env node
 
-var mkdirp = require("mkdirp");
+'use strict';
+
+const parsed = require('../lib/parse-args')(process.argv.slice(2));
+let sources = parsed.sources;
+let shared = parsed.shared;
+
+var mkdirp = require('mkdirp');
```

Comment:
`mkdirp` looks like a new line because it's bumped down, but my policy (lol) was to add es6 style variable casting for the pieces I was working on. I'll change them all one way or another depending on what you two decide _in another PR_ (or you could, if it's a bother?)